registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -687,6 +690,7 @@
     "sha256:5b341f61f50c2e23822b724ef4870a432557b72fc623af27e583fe095bb82655": ["v1.8.1"]
     "sha256:c23cff2c1876c72ffdfd8eb293870f625aae8654af5c8159dffae61496dd5a9f": ["v1.8.10"]
     "sha256:f56da511cdc151e60feaf0c26b01461b54518e9a60a6b62e56b4474a97a69fb7": ["v1.8.11"]
+    "sha256:fc53ffcf05c780e2ba8cd63b6e86b31e0d6f9e55af79eb2f1f042570032f21d7": ["v1.8.12"]
```

Comment:
`fc53ffcf05c7` matches the SHA for kubeadm-control-plane-controller v1.8.12 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:fc53ffcf05c780e2ba8cd63b6e86b31e0d6f9e55af79eb2f1f042570032f21d7/details). 👍🏻